docs/reference/quickstart/aggs-tutorial.asciidoc

```diff
@@ -0,0 +1,530 @@
+[[aggregations-tutorial]]
+== Analyze ecommerce data with aggregations using Query DSL
```

Comment:
Nitpick - I think we should use either `e-commerce` or `eCommerce` and be consistent with whichever we choose.